content/en/opentelemetry/runtime_metrics/java.md

```diff
@@ -0,0 +1,64 @@
+| `jvm.non_heap_memory_init` | The initial Java non-heap memory allocated. | `process.runtime.jvm.memory.init` |
+| `jvm.non_heap_memory_max` | The maximum Java non-heap memory available. | `process.runtime.jvm.memory.limit` |
+| `jvm.gc.old_gen_size` | | `process.runtime.jvm.memory.usage` |
+| `jvm.gc.eden_size` | | `process.runtime.jvm.memory.usage` |
+| `jvm.gc.survivor_size` | | `process.runtime.jvm.memory.usage` |
+| `jvm.gc.metaspace_size` | | `process.runtime.jvm.memory.usage` |
+| `jvm.thread_count` | The number of live threads. | `process.runtime.jvm.threads.count` |
+| `jvm.loaded_classes` | Number of classes currently loaded. | `process.runtime.jvm.classes.loaded` |
+| `jvm.cpu_load.system` | Recent CPU utilization for the whole system. | `process.runtime.jvm.system.cpu.utilization` |
+| `jvm.cpu_load.process` | Recent CPU utilization for the process. | `process.runtime.jvm.cpu.utilization` |
+| `jvm.buffer_pool.direct.used` | Measure of memory used by direct buffers. | `process.runtime.jvm.buffer.usage` |
+| `jvm.buffer_pool.direct.count` | Number of direct buffers in the pool. | `process.runtime.jvm.buffer.count` |
+| `jvm.buffer_pool.direct.capacity` | Measure of total memory capacity of direct buffers. | `process.runtime.jvm.buffer.limit` |
+| `jvm.buffer_pool.mapped.used` | Measure of memory used by mapped buffers. | `process.runtime.jvm.buffer.usage` |
+| `jvm.buffer_pool.mapped.count` | Number of mapped buffers in the pool. | `process.runtime.jvm.buffer.count` |
+| `jvm.buffer_pool.mapped.capacity` | Measure of total memory capacity of mapped buffers. | `process.runtime.jvm.buffer.limit` |
+| `jvm.gc.parnew.time` | The approximate accumulated garbage collection time elapsed. | `process.runtime.jvm.gc.duration` |
+|	`jvm.gc.cms.count` | The total number of garbage collections that have occurred. | N/A |
+|	`jvm.gc.major_collection_count` | The rate of major garbage collections. Set `new_gc_metrics: true` to receive this metric. | N/A |
+|	`jvm.gc.minor_collection_count` | The rate of minor garbage collections. Set `new_gc_metrics: true` to receive this metric. | N/A |
+|	`jvm.gc.major_collection_time` | The fraction of time spent in major garbage collection. Set `new_gc_metrics: true` to receive this metric. | N/A |
+|	`jvm.gc.minor_collection_time` | The fraction of time spent in minor garbage collection. Set `new_gc_metrics: true` to receive this metric. | N/A |
+|	`jvm.os.open_file_descriptors` | | N/A |
+
+
+[1]: /opentelemetry/otel_metrics
+[2]: https://app.datadoghq.com/integrations/java
+[3]: https://opentelemetry.io/docs/instrumentation/java/automatic/
+[4]: https://opentelemetry.io/docs/instrumentation/java/manual/
+[5]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-metrics/runtime-metrics-java8/library
+[6]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-metrics/runtime-metrics-java17/library
```

Comment:
Both of these links 404. Are they the wrong URLs, or will there be something there eventually?

Reply:
Thanks for catching this, the URLs were out of date so I just updated them